study.md

```diff
@@ -67,5 +71,9 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+Router will be seperating URL from the verb and passing them to controller;
+controller with the obtained information will finding the right model; model will
+be reaching the database to get the necessary information and convert into JSON
+and turn it back to controller. Finally, controller will pass it to view which
```

Comment:
We won't be using Rails's views so for our purposes, the controller will only render responses (which may or may not be JSON) to the client.